harkify/test/textnoteservice_setting_test.dart

```diff
@@ -9,13 +9,13 @@ void main() {
   FileSystem fileSystem = MemoryFileSystem();
```

Comment:
What's the difference between a memory filesystem and a filesystem and did you create it as a service?

Reply:
The memory filesystem doesn't create actual files on the server. All files "created" just stay in memory. That makes it perfect for unit testing so it doesn't affect the actual files on the system. Rather than set up a dependency injection system for file systems, I'm using a static variable on my text note service class that defaults to the true file system as a way to switch it over to "test mode" for automated unit tests. I'm sure there's a more elegant DI way to do it, but that is what I came up with given the short timeframe.